src/map/packets/roe_questlog.cpp

```diff
@@ -0,0 +1,35 @@
+﻿/*
+===========================================================================
+
+  Copyright (c) 2020 - Kreidos | github.com/kreidos
```

Comment:
I like individual attribution, but does this have any possible ramifications down the road?
If you had a particularly aggressive employer who tried to claim all your spare time work etc.

Reply:
I'm not sure. (Not a lawyer) I self-attributed based on input from Zircon, and staff. Apparently the FSF deems self-attributing as recommended practice.

Reply:
becomes problematic for future maintainers (I mean project maintainers) we don't want to maintain a list of everyone who ever touched one file on a file by file basis do we? what happens when I pr a change to "your" file here later. We're basically agreeing to topaz having this anyway. I like my attribution handled by the commit log personally, which the only legal ramifications being the license text itself which are there to ensure it stays free as in freedom.